### ISSUE TITLE
Buttons disappear in landscape mode

### ISSUE THREAD

**Issue by arcomer** [Android-L-Porting-Team/Android-L-Mako#55]
when i watch video in landscape mode the menu buttons disappear.
Beta 5 (hebrew language)

**Comment by benpye** [comment]
In what application?

**Comment by arcomer** [comment]
Youtube , chrome
בתאריך 10 ביולי 2014 12:35, "benpye" notifications@github.com כתב:

**Comment by benpye** [comment]
That's correct behaviour, the apps in question enter fullscreen mode in landscape, hiding those buttons when watching video. You can swipe in from the side to bring them into view.

**Comment by arcomer** [comment]
Ok i will test to see what you speaking about
בתאריך 10 ביולי 2014 12:41, "benpye" notifications@github.com כתב:

**Comment by arcomer** [comment]
By the way.... Keep on the great job!!!
בתאריך 10 ביולי 2014 12:43, "arcomer" omer@arcomer.co.il כתב:

**Comment by arcomer** [comment]
When i use the camera in landscape mode i see three dots in the menu bar
but when i prees on them nothing happens
בתאריך 10 ביולי 2014 12:41, "benpye" notifications@github.com כתב:

**Comment by benpye** [comment]
I cannot reproduce that, here the buttons have the correct functionality and again swiping in will show the full icons.
